spec/dummy/app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,7 @@
+html {
```

Comment:
What are the items in this file used for?

Reply:
?

Reply:
This is just for importing bootstrap and setting default overriding values